app/screens/integration_selector/integration_selector.tsx

```diff
@@ -4,6 +4,7 @@
 import React, {useCallback, useEffect, useMemo, useRef, useState} from 'react';
 import {useIntl} from 'react-intl';
 import {View} from 'react-native';
+import {ListItemProps} from 'react-native-elements';
```

Comment:
Are you sure this is the right type?

Reply:
I'll double check

Reply:
It seems just an alias to any, but I'd need to refactor the render items functions to have a good type as properties to its renders. I'll remove it and leave any to fix it.